pyogrio/tests/test_geopandas_io.py

```diff
@@ -460,6 +460,56 @@ def test_write_dataframe(tmp_path, naturalearth_lowres, ext):
     )
 
 
+@pytest.mark.filterwarnings("ignore:.*No SRS set on layer.*")
+@pytest.mark.parametrize("ext", [ext for ext in ALL_EXTS + [".xlsx"] if ext != ".fgb"])
+def test_write_dataframe_nogeom(tmp_path, naturalearth_lowres, ext):
+    """Test writing a dataframe, so without a geometry column.
+
+    FlatGeobuf (.fgb) doesn't seem to support this, and just writes an empty file.
+    """
+    # Prepare test data
+    input_gdf = read_dataframe(naturalearth_lowres)
```

Comment:
```suggestion
    input_df = read_dataframe(naturalearth_lowres).drop(columns=["geometry"])
```

Reply:
I added parameter read_geometry=False, than it doesn't even need to be dropped...